planning.py

```diff
@@ -12,3 +11,114 @@
 import time
 import bisect
 import string
+
+
+class Action(Expr):
+    """Subclasses Expr class to define an Action class that can refer
+    to actions of an existing Node or a HLA"""
+    def isNull(self):
```

Comment:
This doesn't make any sense. If you want to compare a variable, hla, against None, use `hla is None`, not `hla.isNull()`.